docker/build_dockerimage.sh

```diff
@@ -0,0 +1 @@
+docker build healthhacker .
```

Comment:
you forgot -t before healthhacker .
this line should be: docker build -t healthhacker .